frontend/src/pages/add/BookAdd.tsx

```diff
@@ -87,103 +72,88 @@ export default function BookAdd() {
     {
       field: "fromDB",
       header: "Book Status",
-      filterPlaceholder: "Search by Book Status",
       customBody: statusTemplate,
-      cellEditValidator: () => false,
-    },
-    {
-      field: "id",
-      header: "ID",
-      filterPlaceholder: "Search by ID",
-      hidden: true,
-      cellEditValidator: () => false,
+      style: { width: "5%" },
     },
     {
       field: "title",
       header: "Title",
-      filterPlaceholder: "Search by Title",
-      cellEditValidator: () => false,
+      style: { width: "15%" },
     },
     {
       field: "author",
       header: "Authors",
-      filterPlaceholder: "Search by Authors",
-      cellEditValidator: () => false,
+      style: { width: "10%" },
     },
     {
       field: "genres",
       header: "Genre",
-      filterPlaceholder: "Search by Genre",
+      style: { width: "10%" },
       cellEditor: (options) => genreDropdown(options),
     },
     {
       field: "isbn13",
       header: "ISBN 13",
-      filterPlaceholder: "Search by ISBN",
-      cellEditValidator: () => false,
+      style: { width: "10%" },
     },
     {
       field: "isbn10",
       header: "ISBN",
-      filterPlaceholder: "Search by ISBN",
-      cellEditValidator: () => false,
+      style: { width: "10%" },
     },
     {
       field: "publisher",
       header: "Publisher",
-      filterPlaceholder: "Search by Publisher",
-      cellEditValidator: () => false,
+      style: { width: "10%" },
     },
     {
       field: "publishedYear",
       header: "Publication Year",
-      filterPlaceholder: "Search by Publication Year",
-      cellEditValidator: () => false,
+      style: { width: "5%" },
     },
     {
       field: "pageCount",
       header: "Page Count",
-      filterPlaceholder: "Search by Page Count",
-      hidden: true,
+      style: { width: "5%" },
       cellEditValidator: (event: ColumnEvent) =>
         isPositiveInteger(event.newValue),
       cellEditor: (options: ColumnEditorOptions) => numberEditor(options),
     },
     {
       field: "width",
       header: "Width",
-      filterPlaceholder: "Search by Width",
+      style: { width: "5%" },
       cellEditValidator: (event: ColumnEvent) => event.newValue > 0,
       cellEditor: (options: ColumnEditorOptions) => numberEditor(options),
     },
     {
       field: "height",
       header: "Height",
+      style: { width: "5%" },
       filterPlaceholder: "Search by Height",
```

Comment:
I see the mass purge of the filerPlaceholder items and cell editor stuff. Doesn't this loose functionality? Also if you have a way to reintegrate it you may have left some of the stuff undeleted here.

Reply:
All the filters that were deleted were unused, but you're right it looks like I left some residual stuff. I did another check through the columns